.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      strategy:
+        description: Valid semver number <x.x.x> or strategy <patch, minor, major>
+        default: "patch"
+        required: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'pebie' || github.actor == 'Crow-EH' || github.actor == 'fthouraud' || github.actor == 'leguellec' || github.actor == 'rande' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Bump the version using input strategy
+        run: yarn version ${{ github.event.inputs.strategy }}
+
+      - name: Update changelog    
+        id: changelog
+        run: |
+          yarn install
+          CHANGELOG=$(yarn conventional-changelog -p conventionalcommits -r -u 0)
+          echo -e "${CHANGELOG}\n\n\n\n$(cat CHANGELOG.md)" > CHANGELOG.md
+          BODY=$(echo -e "${CHANGELOG}" | sed -e "1,2d")
+          echo "::set-output name=body::$(echo -e "${BODY}")"
+
+      - name: Get version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.1.0
+
+      - name: Create tag
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          git add .
+          git commit -m "Bump - v${{ steps.package-version.outputs.current-version }}"
```

Comment:
How about a conventional message like `chore(something): release v1.2.3` ?

For reference, here's what convention-changelog team does on the release commits : https://github.com/conventional-changelog/conventional-changelog/commit/653101d5937e977339e376dc4da130d3b73d5f4b

Keep in mind they have multiple libraries, we do not need to specify a name in ours.

Reply:
Done